be/src/storage/storage_engine.cpp

```diff
@@ -240,6 +241,15 @@ Status StorageEngine::_open(const EngineOptions& options) {
         return _segment_replicate_executor->get_thread_pool()->num_queued_tasks();
     })
 
+#ifdef USE_STAROS
+    _segment_writer_finalize_executor = std::make_unique<lake::SegmentWriterFinalizeExecutor>();
```

Comment:
need properly clean this executor when doing storage engine stop()/destroy(), do a favor to graceful exit.